geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/data/TableConfCommand.scala

```diff
@@ -61,15 +59,15 @@ class TableConfUpdateCommand extends AccumuloDataStoreCommand {
   override def execute(): Unit = {
     withDataStore { ds =>
       Command.user.info(s"Reading configuration parameters for index '${params.index}'")
-      val table = getTableName(ds, ds.getSchema(params.featureName), params.index)
-      val value = getProp(ds, table, params.key)
-      Command.user.info(s"  current value: ${params.key}=$value")
+      val tables = getTableNames(ds, ds.getSchema(params.featureName), params.index)
+      val values = tables.map(getProp(ds, _, params.key)).distinct
+      values.foreach(v => Command.user.info(s"  current value: ${params.key}=$v"))
 
-      if (params.newValue != value) {
+      if (values != Seq(params.newValue)) {
         Command.user.info(s"Updating configuration parameter to '${params.newValue}'...")
-        setValue(ds, table, params.key, params.newValue)
-        val updated = getProp(ds, table, params.key)
-        Command.user.info(s"  updated value: ${params.key}=$updated")
+        tables.foreach(setValue(ds, _, params.key, params.newValue))
+        val updated = tables.map(getProp(ds, _, params.key)).distinct
+        updated.foreach(v => Command.user.info(s"  updated value: ${params.key}=$v"))
```

Comment:
Hmm... do we want to update all the table confs uniformly?!  Seems like there may be occasional benefits to configuring different partitions differently.

Reply:
hmm, like what?